apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2/TranslationClient.cs

```diff
@@ -358,17 +358,8 @@ public static TranslationClient Create(GoogleCredential credential = null, Trans
             return CreateImpl(scopedCredentials, null, model);
         }
 
-        private static TranslationClient CreateImpl(GoogleCredential scopedCredentials, string apiKey, TranslationModel model)
-        {
-            var service = new TranslateService(new BaseClientService.Initializer
-            {
-                HttpClientInitializer = scopedCredentials,
-                ApplicationName = TranslationClientImpl.ApplicationName,
-                ApiKey = apiKey
-            });
-
-            return new TranslationClientImpl(service, model);
-        }
+        private static TranslationClient CreateImpl(GoogleCredential scopedCredentials, string apiKey, TranslationModel model) =>
```

Comment:
This doesn't appear to use the `model` parameter any more.

Reply:
Eek! Well caught. Will fix :)